pkg/blobserver/google/cloudstorage/storage.go

```diff
@@ -87,6 +89,7 @@ func newFromConfig(_ blobserver.Loader, config jsonconfig.Obj) (blobserver.Stora
 		auth      = config.RequiredObject("auth")
 		bucket    = config.RequiredString("bucket")
 		cacheSize = config.OptionalInt64("cacheSize", 32<<20)
+		ratestr   = config.OptionalString("rate_limit", "10ms") // no faster than 1 req per this duration
```

Comment:
Hah, cacheSize above is without underscore and client_id and client_secret below use an underscore. Great. :)

Reply:
Where does 10ms come from? Is that a Google-published number?

Reply:
Added a comment referencing the relevant GCS docs. Short answer: up to 1000qps for writing and 5000 qps for reading is fine, above that and you're supposed to ramp up to it slowly. The default here is very conservative.